samples/AspNetIntegration/MultipleOutputBindings.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Extensions.Logging;
+
+namespace AspNetIntegration
+{
+    public class MultipleOutputBindings
+    {
+        private readonly ILogger<MultipleOutputBindings> _logger;
+
+        public MultipleOutputBindings(ILogger<MultipleOutputBindings> logger)
+        {
+            _logger = logger;
+        }
+
+        [Function("MultipleOutputBindings")]
+        public MyOutputType Run([HttpTrigger(AuthorizationLevel.Function, "get", "post")] HttpRequest req)
+        {
+            _logger.LogInformation("C# HTTP trigger function processed a request.");
+            var myObject = new MyOutputType
+            {
+                HttpResponse = new OkObjectResult("C# HTTP trigger function processed a request."),
+                Name = "some name"
+            };
+
+            return myObject;
+        }
+
+        public class MyOutputType
+        {
+            [HttpResponseOutput]
+            public IActionResult HttpResponse { get; set; }
+
+            [QueueOutput("functionstesting2", Connection = "AzureWebJobsStorage")]
+            public string Name { get; set; }
+        }
+    }
```

Comment:
Can we set this up as a doc snippet? We can swap the reference to /samples/Extensions/MultiOutput/MultiOutput.cs's "docsnippet_multiple_outputs" in favor of one here.

Reply:
```suggestion
    //<docsnippet_aspnetcore_multiple_outputs>
    public class MultipleOutputBindings
    {
        private readonly ILogger<MultipleOutputBindings> _logger;

        public MultipleOutputBindings(ILogger<MultipleOutputBindings> logger)
        {
            _logger = logger;
        }

        [Function("MultipleOutputBindings")]
        public MyOutputType Run([HttpTrigger(AuthorizationLevel.Function, "post")] HttpRequest req)
        {
            _logger.LogInformation("C# HTTP trigger function processed a request.");
            var myObject = new MyOutputType
            {
                Result = new OkObjectResult("C# HTTP trigger function processed a request."),
                MessageText= "some output"
            };

            return myObject;
        }

        public class MyOutputType
        {
            [HttpResponseOutput]
            public IActionResult Result { get; set; }

            [QueueOutput("myQueue")]
            public string MessageText { get; set; }
        }
    }
    //</docsnippet_aspnetcore_multiple_outputs>
```